<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorPageScriptStats</class>
 <widget class="QWidget" name="ScriptEditorPageScriptStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="list"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMoveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMoveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ReachStringPicker" name="name" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="sortOrder">
       <item>
        <property name="text">
         <string>Ascending</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ignored</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Descending</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unused 02</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="format">
       <item>
        <property name="text">
         <string>Number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Number with Sign</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Percentage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="groupByTeam">
       <property name="text">
        <string>Group Players by Team</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sort Order</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReachStringPicker</class>
   <extends>QWidget</extends>
   <header>ui/widgets/string_picker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
